### PR TITLE
Adding Token lifetime content and links

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Adding Token lifetime content and links
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-368946](https://oktainc.atlassian.net/browse/OKTA-368946)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 1]
```diff
@@ -1096,6 +1096,20 @@ This section contains some general information about claims, as well as detailed
...
+When you are using the [Okta Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), the lifetime of the JWT tokens is hard-coded to the following values:
+
+* **ID token:** 60 minutes
+* **Access token:** 60 minutes
+* **Refresh token:** 90 days
+
+When you are using a [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), you can configure the lifetime of the JWT tokens:
+
+* **ID token:** At least five minutes, no more than 60 minutes
+* **Access tokens:** At least five minutes, no more than 24 hours (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)). See the [Authorization Servers API](/docs/reference/api/authorization-servers/#actions-object).
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:1110]: The help link does not work. Also do we need to link both prod and api doc here?

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 2]
```diff
@@ -1096,6 +1096,20 @@ This section contains some general information about claims, as well as detailed
...
+* **Refresh tokens:** At least 10 minutes, maximum of five years (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)). See the [Authorization Servers API](/docs/reference/api/authorization-servers/#actions-object).
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:1111]: The refresh token has lifetime and idle time(window), `At least 10 minutes, maximum of five years` is for the idle time only.

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:1110]: The help link should work now - they were working on it from the prod side this morning. Which would you rather have? The link to how to configure the timeframe for the token in the access policy or the authorization server api information?

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 3]
```diff
@@ -1096,6 +1096,20 @@ This section contains some general information about claims, as well as detailed
...
+When you are using the [Okta Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), the lifetime of the JWT tokens is hard-coded to the following values:
+
+* **ID token:** 60 minutes
+* **Access token:** 60 minutes
+* **Refresh token:** 90 days
+
+When you are using a [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), you can configure the lifetime of the JWT tokens:
+
+* **Access tokens:** At least five minutes, no more than 24 hours (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)).
+* **Refresh tokens:** The minimum active token lifetime, which is five minutes. The idle time window is at least 10 minutes, with a maximum of five years (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)).
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:1110]: The minimum lifetime is the same as the access token lifetime. `, which is five minutes` can be removed, if the admin set the access token as 10 minutes, then the refresh token minimum lifetime is 10 minutes.

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 4]
```diff
@@ -1096,6 +1096,20 @@ This section contains some general information about claims, as well as detailed
...
+* **ID tokens:** Not configurable
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:1111]: it should also mention the lifetime is 60 minutes.

**Review by yuliu-okta — APPROVED** [review]

**Pull request merged**